spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 require "alaveteli_features/spec_helpers"
 require 'strip_attributes/matchers'
 
-require_relative 'spec_coverage'
+require_relative 'spec_coverage' if ENV['COVERAGE'] == 'local' || ENV['CI']
```

Comment:
I think:

```diff
- Don't load simplecov if when not checking coverage
+ Don't load simplecov when not checking coverage
```

I also think the commit message should explain _why_ this change was necessary, as I don't really understand the problem based on the change alone. Something to do with GH actions I know; I thought it was that we didn't want to run coverage when running specs (needed separate step), but then `|| ENV['CI']` makes it look like we _do_ want to load this file when running specs on CI?

Reply:
Update the commit message